core/templates/dev/head/services/SpeechSynthesisChunkerService.js

```diff
@@ -112,18 +120,22 @@ oppia.factory('SpeechSynthesisChunkerService', [
       // Convert links into speakable text by extracting the readable value.
       elt.find('oppia-noninteractive-' + RTE_COMPONENT_NAMES.Link)
         .replaceWith(function() {
-          if (this.attributes['text-with-value'] !== undefined) {
-            return this.attributes['text-with-value'].textContent
-              .replace(/&quot;/g, '');
+          if ((<HTMLElement><any>this).attributes[
+            'text-with-value'] !== undefined) {
+            return (<HTMLElement><any>this).attributes[
+          'text-with-value'].textContent.replace(/&quot;/g, '');
```

Comment:
Indent by 4 more spaces (2 more than the previous line).

Reply:
The linter is disabled. I will enable the linter once tslint stuff is done. You can ignore the linter erros till then.